irohad/consensus/yac/storage/impl/yac_block_storage.cpp

```diff
@@ -15,12 +15,7 @@
  * limitations under the License.
  */
 #include "consensus/yac/storage/yac_block_storage.hpp"
-
-#include <algorithm>
-#include <utility>
-
-#include "consensus/consensus_common.hpp"
-#include "consensus/yac/messages.hpp"
```

Comment:
algorithm, utility and consensus/yac/messages.hpp should be here due to iwyu principle